src/content/docs/kubernetes-pixie/kubernetes-integration/uninstall-kubernetes.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: 'Uninstall Kubernetes integration'
+tags:
+  - Integrations
+  - Kubernetes integration
+  - Uninstall
+metaDescription: "Learn how to uninstall Kubernetes integration."
+---
+
+The New Relic Kubernetes integration can be easly uninstalled from your platform. 
+
+Ensure you are operating in the appropriate context on the machine where you are running Helm and `kubectl`. You can check the available contexts with this command:
+
+```shell
+kubectl config get-contexts
+```
+
+And switch to the desired context using:
+
+```shell
```

Comment:
@nbaenam we have retired the <var> tags because they don't work when we add a language (in this case, "shell" to the code snippet fence.  We can no longer highlight individual words--only lines--so in this case the line highlighting won't help. I would just remove the <var>asd</var> tags.

Reply:
Thanks, @rhetoric101!